src/app/manage/create-new-user/create-new-user.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="container">
+    <form class="container" [formGroup]="createForm" class="form-group">
+        <div class="form-group">
+            First Name <input type="text" formControlName="firstName" class="form-control" />
+        </div>
+        <div class="form-group">
+            Last Name <input type="text" formControlName="lastName" class="form-control" />
+        </div>
+        <div class="form-group">
+            Login <input type="text" formControlName="login" class="form-control" />
+        </div>
+        <div class="form-group">
+            Password <input type="text" formControlName="password" class="form-control" />
+            <div class="alert alert-danger" *ngIf="createForm.get('password').invalid && createForm.get('password').touched">
```

Comment:
does it make sense to move this condition to the component? If any condition is big try to break them into readable pieces or move them to components